pyproject.toml

```
@@ -4,17 +4,17 @@
 
 [project]
 name = "brainscore_core"
-version = "0.1.0"
+version = "2.1.0"
 description = ""
 authors = []
 license = { 'file' = 'LICENSE' }
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.8"
 
 dependencies = [
     "tqdm",
-    "numpy>=1.21",
-    "brainio@git+https://github.com/brain-score/brainio.git@main",
+    "numpy>=1.21,<2.0",
+    "brainio@git+https://github.com/brain-score/brainio.git@sw/python_3-11_migration",
```

```suggestion
    "brainio@git+https://github.com/brain-score/brainio.git@main",
```

Needs to be changed back once BrainIO is updated.

---

This has been resolved in-person. TL;DR referencing 3.11 branch until the migration is complete to avoid things breaking. Then make a separate commit to change it to main.